docs/Guides/Getting-Started.md

```diff
@@ -294,12 +336,7 @@ async function routes (fastify, options) {
   })
 
   fastify.get('/animals/:animal', async (request, reply) => {
-    const result = await collection.findOne({ animal: request.params.animal })
-    if (!result) {
-      throw new Error('Invalid value')
-    }
-    return result
-  })
+    const rer
```

Comment:
Where'd this go?

Reply:
Sorry I might have removed it by mistake.. I have reverted these changes